adapter/config/default_config.go

```diff
@@ -124,6 +124,16 @@ var defaultConfig = &Config{
 			DNS: upstreamDNS{
 				DNSRefreshRate: 5000,
 				RespectDNSTtl:  false,
+				DNSResolver: dnsResolverConfig{
```

Comment:
this is the default one right? I don't see resolver type here.

Reply:
Yeah, when it is empty, we don't set a resolver